src/prdoc.rs

```diff
@@ -156,6 +156,11 @@ fn validate(args: &Args, prdoc: &Prdoc, w: &Workspace) -> Result<()> {
     writeln!(stdout, "validating prdocs...")?;
     let prdocs = get_prdocs(w, &prdoc.prdoc_path, false, &prdoc.crates)?;
 
+    if let Some(max_allowed_bump) = prdoc.max_allowed_bump {
```

Comment:
I don't think this should be a seperate part. It should be a part of the end report that contains all the information.

I also think we should apply the limit on the actual semver change not the change the PRDoc claims.

Reply:
Then maybe we output `max(predicted, prdoc)`